en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/M/Mislead.md

```diff
@@ -1,5 +1,20 @@
 #### Mislead
-<!-- TODO Check and tag this spell -->
+<!-- markdownlint-disable link-image-reference-definitions -->
+[_metadata_:spell_name]:- "Mislead"
+[_metadata_:spell_level]:- "5"
+[_metadata_:spell_school]:- "illusion"
+[_metadata_:ritual]:- "false"
+[_metadata_:casting_time_amount]:- "1"
+[_metadata_:casting_time_unit]:- "action"
+[_metadata_:range]:- "Self"
+[_metadata_:target]:- "Self"
+[_metadata_:components_verbal]:- "false"
+[_metadata_:components_somatic]:- "true"
+[_metadata_:components_material]:- "false"
+[_metadata_:duration]:- "1 hour"
+[_metadata_:concentration]:- "true"
+[_metadata_:compared_to_wotc_srd_5.1]:- "mechanics_different_wording_different"
```

Comment:
The mechanical difference here is this:
> The copy is revealed as an illusion with any physical interaction, as solid objects and creatures pass through it.

The WotC version doesn't have this or anything similar; it does not state how the illusion could be identified as such, while other illusion spells do specify that. I think this is a good addition since it makes it very clear. With the WotC version an argument could be made that the illusion can not be identified by physical interaction, making it potentially mechanically different.